src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1643,18 +1643,16 @@ constexpr bool has_argument_tags_v = has_argument_tags<F>::value;
  * DataBox, `box`
  */
 template <typename TagsList, typename F, typename BoxTags, typename... Args,
-          Requires<not DataBox_detail::has_argument_tags_v<std::decay_t<F>>> =
-              nullptr>
+          std::enable_if_t<not DataBox_detail::has_argument_tags_v<
```

Comment:
Ah sorry, this should be a `Requires`. `Requires` is `enable_if` that gives nice-ish error messages on older compilers. It was just in the other case where we don't need `Requires` at all. Does that make sense? I'm happy for you to squash :)